pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/KitSubKits.java

```diff
@@ -63,10 +64,10 @@ public static List<KitSubKits> getSubKits(@NonNull String realm, @NonNull String
         });
 
         if (results.resultException != null) {
-            throw new RuntimeException("Error looking up kit type  ", results.resultException);
+            throw new DsmInternalError("Error looking up sub kit with type  "+ kitType, results.resultException);
         }
         if (subKits.size() > 1) { //otherwise it found the normal kit
-            logger.info("Found " + subKits.size() + " subKits");
+            logger.info("Found {} subKits", subKits.size());
```

Comment:
Maybe add "for `realm` and `kitTypeName`" here so that the logs are more informative--if we even need this as an info.  Might be better as a debug.